server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/tservers.ftl

```diff
@@ -18,39 +18,15 @@
     under the License.
 
 -->
-      <div class="row">
+    <div class="row">
         <div class="col-xs-12">
-          <h3>${title}</h3>
+            <h3>Tablet Server Overview</h3>
         </div>
-      </div>
+    </div>
```

Comment:
looks like this div is out of line with the others. also the `<h3>` has one or two extra spaces added.